Workade/Managers/MagazineTransitionManager.swift

```diff
@@ -0,0 +1,224 @@
+    private var duration: Double {
+        return isPresent ? 0.75 : 0.65
+    }
+    private var cellLabelHeight: CGFloat?
+    
+    private let blurEffectView: UIView = {
+        let blurEffect = UIBlurEffect(style: .dark)
+        let blurEffectView = UIVisualEffectView(effect: blurEffect)
+        blurEffectView.translatesAutoresizingMaskIntoConstraints = false
+        blurEffectView.reservations = [.init(.alpha, from: 0, to: 1, animated: true)]
+        
+        return blurEffectView
+    }()
+    
+    private func makeTitleImageView(origin view: MagazineTitleImageView) -> MagazineTitleImageView {
+        let magazine = view.magazine
+        let copyTitleImageView = MagazineTitleImageView(by: magazine)
+        cellTextLabel.text = copyTitleImageView.magazine.title
+        copyTitleImageView.backgroundColor = .theme.groupedBackground
+        copyTitleImageView.reservations = [.init(.cornerRadius, from: 16, to: 0)]
+        copyTitleImageView.titleLabel.reservations = [.init(.alpha, from: 0, to: 1, animated: isPresent)]
+        copyTitleImageView.bookmarkButton.reservations = [.init(.alpha, from: 0, to: 1, animated: isPresent)]
+        
+        return copyTitleImageView
+    }
+    
+    private let whiteView: UIView = {
+        let view = UIView()
+        view.backgroundColor = .theme.background
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.reservations = [.init(.cornerRadius, from: 16, to: 0)]
+        
+        return view
+    }()
+    
+    private let bookmarkButton: UIButton = {
+        let button = UIButton()
+        button.setImage(SFSymbol.bookmark.image, for: .normal)
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.reservations = [.init(.alpha, from: 1, to: 0, animated: false)]
+        
+        return button
+    }()
+
+    private let cellTextLabel: UILabel = {
+        let label = UILabel()
+        label.font = .customFont(for: .caption2)
+        label.textColor = .white
+        label.numberOfLines = 0
+        label.lineBreakMode = .byWordWrapping
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.reservations = [.init(.alpha, from: 1, to: 0, animated: false)]
+        
+        return label
+    }()
+}
+
+extension MagazineTransitionManager: UIViewControllerAnimatedTransitioning {
+    func transitionDuration(using transitionContext: UIViewControllerContextTransitioning?) -> TimeInterval {
+        return duration
+    }
+    
+    func animateTransition(using transitionContext: UIViewControllerContextTransitioning) {
+        let containerView = transitionContext.containerView
+        containerView.subviews.forEach { $0.removeFromSuperview() }
+        blurEffectView.frame = containerView.frame
+        
+        let fromVC = transitionContext.viewController(forKey: .from)
+        let toVC = transitionContext.viewController(forKey: .to)
```

Comment:
[P3]
```suggestion
        let fromViewController = transitionContext.viewController(forKey: .from)
        let toViewController = transitionContext.viewController(forKey: .to)
```

수정 부탁드립니다!